src/test/java/com/klarna/hiverunner/ReservedKeywordTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2013 Klarna AB
```

Comment:
We should use Apache 2.0. I'm pretty sure I just copied those docs from some apache template a long time ago. There might be newer ones available?